eotsmanager/cmd/eotsd/daemon/pop.go

```diff
@@ -0,0 +1,141 @@
+package daemon
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cometbft/cometbft/crypto/tmhash"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	bbn "github.com/babylonchain/babylon/types"
+	btcstktypes "github.com/babylonchain/babylon/x/btcstaking/types"
+	"github.com/babylonchain/finality-provider/eotsmanager"
+	"github.com/babylonchain/finality-provider/eotsmanager/config"
+	"github.com/babylonchain/finality-provider/log"
+	"github.com/urfave/cli"
+)
+
+// PoPExport the data for exporting the PoP.
+// The PubKeyHex is the public key of the finality provider BTC key to load
+// the private key and sign the AddressSiged.
+type PoPExport struct {
+	PubKeyHex     string                           `json:"pub_key_hex"`
+	PoP           btcstktypes.ProofOfPossessionBTC `json:"pop"`
+	PoPHex        string                           `json:"pop_hex"`
+	AddressSigned string                           `json:"address_signed"`
+}
+
+var ExportPoPCommand = cli.Command{
+	Name:      "pop-export",
+	Usage:     "Exports the Proof of Possession by Signing over the finality provider address with the EOTS private key.",
+	UsageText: "pop-export [bbn-address]",
+	Description: `Parse the address received as argument, hash it with
+	sha256 and sign based on the Schnorr key associated with the key-name or btc-pk flag.
```

Comment:
```suggestion
	sha256 and sign based on the EOTS key associated with the key-name or btc-pk flag.
```

I think we use Schnorr/BTC/EOTS/FinalityProvider key interchangeably, which might be confusing for users. Maybe we just say EOTS key here

Reply:
https://github.com/babylonchain/finality-provider/commit/d1c66761d2fbe7d9ca9ab7ddb472ccb3db28cc26